src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectFile.cs

```diff
@@ -54,12 +54,12 @@ public string GetPropertyValue(string name)
 
         protected class BuildResult
         {
-            public readonly MSB.Execution.BuildResult Result;
+            public readonly bool WasBuilt;
             public readonly MSB.Execution.ProjectInstance Instance;
 
-            internal BuildResult(MSB.Execution.BuildResult result, MSB.Execution.ProjectInstance instance)
+            internal BuildResult(bool wasBuilt, MSB.Execution.ProjectInstance instance)
             {
-                this.Result = result;
+                WasBuilt = wasBuilt;
```

Comment:
I would either use 'this.WasBuilt' here or remove 'this.' from the line below.  I'm ambivalent to which but would prefer the consistency.